packages/pages/src/dev/server/ssr/findMatchingModule.ts

```diff
@@ -30,3 +35,86 @@ export const findModuleInternal = async (
 
   return null;
 };
+
+export const loadModuleInternalFromFilepath = async (
+  devserver: ViteDevServer,
+  moduleFilePath: string
+): Promise<ModuleInternal> => {
+  const moduleModule = await loadViteModule<ModuleModule>(
+    devserver,
+    moduleFilePath
+  );
+
+  return convertModuleToModuleInternal(moduleFilePath, moduleModule);
+};
+
+export const getPostCssPathFromModuleName = async (
+  moduleName: string | undefined,
+  projectStructure: ProjectStructure
+): Promise<string> => {
+  if (moduleName == undefined) {
+    return "";
+  }
+  const moduleFilePaths =
+    getModuleFilepathsFromProjectStructure(projectStructure);
+  const project = createTsMorphProject();
+  let targetModuleFilePath = "";
+  for (const moduleFilePath of moduleFilePaths) {
+    project.addSourceFileAtPath(moduleFilePath);
+    const sourceFile = project.getSourceFile(moduleFilePath);
+    if (sourceFile !== undefined) {
+      const exportedDeclarations = sourceFile.getExportedDeclarations();
+      for (const [name, declarations] of exportedDeclarations) {
+        if (name == "config") {
+          declarations.forEach((declaration) => {
+            for (const child of declaration.getChildren()) {
+              const text = child.getText();
+              if (text.includes("name:")) {
+                const parsedName = text
+                  .substring(
+                    text.indexOf('"') + 1,
```

Comment:
What if someone uses single quotes? I think we've done a less hacky way of this before. Can't we just get all of the config and convert it to json?

Reply:
It doesn't convert to json very well. The name we're looking for looks something like this and there's also other code in the file:

export const config: ModuleConfig = {
  name: "prefix"
}

I'll update to handle all quote types: " ` and '

Reply:
I thought there was still a better. Way. Talk to Alexis as she may know. If there's not then we can go with this.